src/api/helpers/string/index.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * isEmptyString
+ * Check if a string is empty
+ * @param {String}
+ * @returns {Boolean}
+ */
+export const isEmptyString = (str: string) => str === '';
```

Comment:
Do you want to `str.trim()`?

Reply:
We could, but would like to do a separate PR so that we can track and test it properly and progress this PR. Have created EMS-4103 to tackle this.